.gitignore

```diff
@@ -20,9 +20,9 @@ dist/assets
 dist/*.worker.*
 
 # tmp folder for build and electron
-.app
```

Comment:
You messed this up somehow, these aren't ignored anymore.

Reply:
they had a space in front of them somehow, fixed it